modules/RTC/RTCUtils.js

```diff
@@ -811,17 +764,9 @@ class RTCUtils extends Listenable {
                 const getUserMedia
                     = navigator.webkitGetUserMedia.bind(navigator);
 
-                if (navigator.mediaDevices) {
-                    this.getUserMedia = wrapGetUserMedia(getUserMedia);
-                    this.enumerateDevices
-                        = wrapEnumerateDevices(
-                            navigator.mediaDevices.enumerateDevices.bind(
-                                navigator.mediaDevices));
-                } else {
```

Comment:
We are not supporting anymore browsers where GUM is with callbacks or this is removed by mistake?